src/Makefile

```diff
@@ -922,46 +557,24 @@ FLATBUFFERS_LIB = $(EXTERNAL_FLATBUFFERS)build/libflatbuffers.a
 LZMA_LIB = $(EXTERNAL_LZMA)build/liblzma.a
 LUA_LIB = ${EXTERNAL_LUA}install/lib/liblua.a
 
-EXTERNAL_LIBS := $(JSON_LIB) $(ZLIB_LIB) $(OPENSSL_LIB) $(CRYPTO_LIB) $(SQLITE_LIB) $(LIBYAML_LIB) $(LIBPCRE2_LIB)
-
-# Adding libcurl only on Windows, Linux and MacOS
-ifeq (${TARGET},winagent)
-	EXTERNAL_LIBS += $(LIBCURL_LIB)
-else ifeq (${uname_S},Linux)
-	EXTERNAL_LIBS += $(LIBCURL_LIB)
-else ifeq (${uname_S},Darwin)
-	EXTERNAL_LIBS += $(LIBCURL_LIB)
-endif
+EXTERNAL_LIBS := $(JSON_LIB) $(ZLIB_LIB) $(OPENSSL_LIB) $(CRYPTO_LIB) $(SQLITE_LIB) $(LIBYAML_LIB) $(LIBPCRE2_LIB) $(LIBCURL_LIB) $(LZMA_LIB) $(MINIZIP_LIB) $(MSGPACK_LIB) $(LIBFFI_LIB) $(BZIP2_LIB)
 
 # Adding libraries only for server
-ifeq (${TARGET},server)
-	EXTERNAL_LIBS_NO_WHOLE += $(ROCKSDB_LIB) $(FLATBUFFERS_LIB)
-	EXTERNAL_LIBS += $(LZMA_LIB) $(MINIZIP_LIB)
-endif
+EXTERNAL_LIBS_NO_WHOLE += $(ROCKSDB_LIB) $(FLATBUFFERS_LIB)
 
-ifneq (${TARGET},winagent)
-EXTERNAL_LIBS += $(MSGPACK_LIB)
-ifneq (${TARGET},agent)
-EXTERNAL_LIBS += $(LIBFFI_LIB) $(BZIP2_LIB)
-endif
 ifeq (${uname_S},Linux)
 ifneq ($(CHECK_CENTOS5),YES)
 EXTERNAL_LIBS +=  ${LUA_LIB} ${RPM_LIB} ${POPT_LIB}
 endif
 ifneq (,$(filter ${USE_AUDIT},YES yes y Y 1))
 EXTERNAL_LIBS += ${AUDIT_LIB}
 endif
-
 ifeq ($(CHECK_CENTOS5),YES)
```

Comment:
The CentOS 5 conditions should also be removed, as they are agent-only:
- https://documentation.wazuh.com/current/installation-guide/packages-list.html
```suggestion
```